qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1393,6 +1393,39 @@ def substitute_node(self, node, op, inplace=False):
             self._decrement_op(node.op)
         return new_node
 
+    def separable_circuits(self) -> List["DAGCircuit"]:
+        """Separate the dag circuit into its weakly connected components.
```

Comment:
Similarly here, an additional description of what these means in circuit language would be helpful.